classification2.Rmd

```diff
@@ -100,29 +105,31 @@ using a larger test data set). Here, we will use 75% of the data for training,
 and 25% for testing.  
 
 The `initial_split` function from `tidymodels` handles the procedure of splitting
-the data for us. It also takes two very important steps when splitting to ensure
+the data for us. It also applies two very important steps when splitting to ensure
 that the accuracy estimates from the test data are reasonable. First, it
-**shuffles** the data before splitting. This ensures that any ordering present
+**shuffles** the data before splitting, which ensures that any ordering present
 in the data does not influence the data that ends up in the training and testing sets.
 Second, it **stratifies** the data by the class label, to ensure that roughly
-the same proportion of each class ends up in both the training and testing sets. For example, if roughly 65% of the
-observations are from the benign class (`B`) and 35% are from the malignant class (`M`),
-then `initial_split` ensures that roughly 65% of the training data are benign, 
-35% of the training data are malignant,
+the same proportion of each class ends up in both the training and testing sets. For example,
+in our data set, roughly 63% of the
+observations are from the benign class (`B`), and 37% are from the malignant class (`M`),
+so `initial_split` ensures that roughly 63% of the training data are benign, 
+37% of the training data are malignant,
 and the same proportions exist in the testing data.
 
 Let's use the `initial_split` function to create the training and testing sets.
 We will specify that `prop = 0.75` so that 75% of our original data set ends up
 in the training set. We will also set the `strata` argument to the categorical label variable
-(here, `Class`) to ensure that the training and validation subsets contain the
+(here, `Class`) to ensure that the training and testing subsets contain the
 right proportions of each category of observation.
 The `training` and `testing` functions then extract the training and testing
 data sets into two separate data frames.
+
 ```{r 06-initial-split}
 set.seed(1)
 cancer_split <- initial_split(cancer, prop = 0.75, strata = Class)
 cancer_train <- training(cancer_split)
-cancer_test <- testing(cancer_split)
+cancer_test <- testing(cancer_split) 
 ```
 
```

Comment:
should emphasize again that in a typical data analysis, you set.seed once at the beginning.

Reply:
(below -- github didn't let me comment the relevant line)

Reply:
as per Trevor's comment this is going to be addressed in global set.seed issue